main.py

```diff
@@ -0,0 +1,11 @@
+import random 
+
+if __name__ == "__main__":
+   while True:     
+      print(''' 1. roll the dice             2. exit     ''')    
+      user = int(input("what you want to do\n"))     
+      if user==1:         
+         number = random.randint(1,6)         
+         print(number)     
+      else:         
```

Comment:
The program will exit if the user gives an invalid input (i.e. 'roll'). Is this the intended behavior?